include/secp256k1.h

```diff
@@ -40,6 +40,26 @@ extern "C" {
 #  define SECP256K1_ARG_NONNULL(_x)
 # endif
 
+# if defined _MSC_VER || defined __CYGWIN__
+#  define SECP256K1_HELPER_DLL_IMPORT __declspec(dllimport)
```

Comment:
I'm not sure this is the case. This is an old reference, but I have to assume it still holds:

> You do not need to use __declspec(dllimport) for your code to compile correctly, but doing so allows the compiler to generate better code.

https://msdn.microsoft.com/en-us/library/aa271769(v=vs.60).aspx